test/models/user_test.rb

```diff
@@ -4,4 +4,9 @@ class UserTest < ActiveSupport::TestCase
   # test "the truth" do
   #   assert true
   # end
+
+  test "cannot create user without a name" do
```

Comment:
Saving and creating a user are different (as seen below). Change to "create" to "save"

Reply:
Also, we do not have "name" in the database. Change "name" to "username".